src/wallet/wallet_rpc_server_commands_defs.h

```diff
@@ -2793,5 +2793,25 @@ namespace wallet_rpc
     };
     typedef epee::misc_utils::struct_init<response_t> response;
   };
+
+  struct COMMAND_RPC_GET_DEFAULT_FEE_PRIORITY
+  {
+    struct request_t
+    {
+      BEGIN_KV_SERIALIZE_MAP()
+      END_KV_SERIALIZE_MAP()
+    };
+    typedef epee::misc_utils::struct_init<request_t> request;
+
+    struct response_t
+    {
+      uint64_t priority;
```

Comment:
```suggestion
      uint32_t priority;
```

Is there a reason for `uint64_t` here? Otherwise I would also adjust it.